tests/test_helpers.py

```diff
@@ -27,23 +27,23 @@
 from telegram import MessageEntity
 from telegram.message import Message
 from telegram.utils import helpers
-from telegram.utils.helpers import _UtcOffsetTimezone, _datetime_to_float_timestamp
+from telegram.utils.helpers import _datetime_to_float_timestamp
 
 
 # sample time specification values categorised into absolute / delta / time-of-day
-ABSOLUTE_TIME_SPECS = [dtm.datetime.now(tz=_UtcOffsetTimezone(dtm.timedelta(hours=-7))),
+ABSOLUTE_TIME_SPECS = [dtm.datetime.now(tz=dtm.timezone(dtm.timedelta(hours=-7))),
                        dtm.datetime.utcnow()]
 DELTA_TIME_SPECS = [dtm.timedelta(hours=3, seconds=42, milliseconds=2), 30, 7.5]
-TIME_OF_DAY_TIME_SPECS = [dtm.time(12, 42, tzinfo=_UtcOffsetTimezone(dtm.timedelta(hours=-7))),
+TIME_OF_DAY_TIME_SPECS = [dtm.time(12, 42, tzinfo=dtm.timezone(dtm.timedelta(hours=-7))),
                           dtm.time(12, 42)]
 RELATIVE_TIME_SPECS = DELTA_TIME_SPECS + TIME_OF_DAY_TIME_SPECS
 TIME_SPECS = ABSOLUTE_TIME_SPECS + RELATIVE_TIME_SPECS
 
 
-class TestHelpers(object):
+class TestHelpers:
     def test_escape_markdown(self):
         test_str = '*bold*, _italic_, `code`, [text_link](http://github.com/)'
-        expected_str = '\*bold\*, \_italic\_, \`code\`, \[text\_link](http://github.com/)'
+        expected_str = r'\*bold\*, \_italic\_, \`code\`, \[text\_link](http://github.com/)'
```

Comment:
Why do we want the `r` here? Stupid me asking stupid questions …

Reply:
As far as I see it we strictly speaking try to escape `*`, `_` etc however these are not actually proper escape sequences. Python does handle this correctly though I am not sure if this is defined behaviour and of the two alternatives (either `\\` or r-tagging) I think r-tagging is better. Not sure however If this is necessary for python 3...